model_compression_toolkit/core/common/mixed_precision/mixed_precision_quantization_config.py

```diff
@@ -38,11 +53,12 @@ class MixedPrecisionQuantizationConfig:
 
     Args:
         compute_distance_fn (Callable): Function to compute a distance between two tensors. If None, using pre-defined distance methods based on the layer type for each layer.
-        distance_weighting_method (MpDistanceWeighting): MpDistanceWeighting enum value that provides a function to use when weighting the distances among different layers when computing the sensitivity metric.
+        distance_weighting_method (MpDistanceWeighting): distance weighting method to use. By default, MpDistanceWeighting.AVG.
         num_of_images (int): Number of images to use to evaluate the sensitivity of a mixed-precision model comparing to the float model.
         configuration_overwrite (List[int]): A list of integers that enables overwrite of mixed precision with a predefined one.
         num_interest_points_factor (float): A multiplication factor between zero and one (represents percentage) to reduce the number of interest points used to calculate the distance metric.
-        use_hessian_based_scores (bool): Whether to use Hessian-based scores for weighted average distance metric computation.
+        use_hessian_based_scores (bool): Whether to use Hessian-based scores for weighted average distance metric
+          computation. This flag is going to be deprecated, use distance_weighting_method.HESSIAN instead.
```

Comment:
Should we add a warning about this deprecation?

Reply:
Remove deprecation from the doc. I don't know yet what we are going to do with this and anyway it is not very important because I don't believe anyone uses it